src/Security/UserChecker.php

```diff
@@ -91,7 +91,7 @@ private function checkIfAccountIsLocked(User $user): void
      */
     private function checkIfAccountIsDisabled(User $user): void
     {
-        if (false !== $user->isEnabled()) {
+        if (!$user->disable) {
```

Comment:
What about `(!$user->start || $user->start < $time) && (!$user->stop || $user->stop > $time)`?

Reply:
That's already handled in https://github.com/contao/core-bundle/pull/1256/commits/e534341f23d6aae78e43a492c4f34c51627fb3cb#diff-3965f81a275b864b36b992503eea7bd5R126, apparently because the log messages are different.